aiida_zeopp/parsers/network.py

```diff
@@ -61,22 +61,17 @@ def parse(self, **kwargs):
 
             with self.retrieved.open(fname, 'rb') as handle:
                 if parser is None:
-
-                    # just add file, if no parser implemented
-                    parsed = SinglefileData(file=handle)
-                    self.out(link, parsed)
-
-                    # workaround: if block pocket file is empty, raise an error
-                    # (it indicates the calculation did not finish)
                     if link == 'block':
                         content = handle.read()
-
                         if not content.strip():
```

Comment:
```suggestion
                        if not parsed.content().strip():
```

Reply:
Thanks @ltalirz 
When I got the error, I checked the content of `content` and realized it is empty and therefore the code raises an exception. 
Yeah, it is a better solution.